src/features/expense/ui/UnReviewedExpenseView/UnReviewedExpenseView.test.tsx

```
@@ -0,0 +1,61 @@
+  },
+});
+
+const wrapper = ({ children }: { children: React.ReactNode }) => (
+  <QueryClientProvider client={queryClient}>{children}</QueryClientProvider>
+);
+
+const mockCount = expenseHooks.useExpenseCountByRange as ReturnType<
+  typeof vi.fn
+>;
+const mockList = expenseHooks.useExpensesByRange as ReturnType<typeof vi.fn>;
+const mockUpdate = expenseHooks.useUpdateExpense as ReturnType<typeof vi.fn>;
```

ESLint

[error] 'mockUpdate' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)